kDriveCore/Data/UploadQueue/Operation/UploadOperation.swift

```diff
@@ -422,6 +445,30 @@ public final class UploadOperation: AsynchronousOperation, UploadOperationable,
         SentryDebug.uploadOperationCleanSessionBreadcrumb(uploadFileId)
 
         let cleanFileClosure: (UploadFile) -> Void = { file in
+            // Clean the remote session, if valid. Invalid ones are already gone server side.
+            let driveId = file.driveId
+            let userId = file.userId
+            let uploadingSession = file.uploadingSession?.detached()
+            self.enqueue {
```

Comment:
Do we really want to strongly retain self ?

Reply:
Every use of `enqueue` `enqueueCatching` is making a strong reference on self in master.  (I changed that in the experimental upload from memory branch without much success solving my actual leaks _on this experimental branch_)

We could switch to [weak self] everywhere if you want. I do not see a reason to do so as I do not see retain cycles. (the UploadOperation is released when the upload is complete)

Should I use [weak self] everywhere in `enqueue` `enqueueCatching` you think ?